src/picongpu/include/particles/particleToGrid/derivedAttributes/LarmorEnergy.hpp

```diff
@@ -55,13 +55,16 @@ namespace derivedAttributes
         const float_X gamma2 = gamma * gamma;
         const float_X c2 = SPEED_OF_LIGHT * SPEED_OF_LIGHT;
 
-        const float3_X mom_dt = mom - mom_mt1;
+        const float3_X mom_dt = (mom - mom_mt1) / float_X(DELTA_T);
         const float_X el_factor = charge * charge
             / (float_X(6.0) * PI * EPS0 *
-               c2 * c2 * SPEED_OF_LIGHT * mass * mass);
-        const float_X energyLarmor = el_factor * gamma2 * gamma2
-            * (math::abs2(mom_dt) -
-               math::abs2(math::cross(mom, mom_dt)));
+               c2 * SPEED_OF_LIGHT * mass * mass) * gamma2 * gamma2;
+        const float_X momentumToBetaConvert = float_X(1.0)/ (mass * SPEED_OF_LIGHT * gamma);
+        const float_X energyLarmor = el_factor
+                                     * ( math::abs2(mom_dt)
+                                         - momentumToBetaConvert * momentumToBetaConvert
```

Comment:
that will be very inprecise for relativistic particles.
can't your formulate it in terms of beta*gamma?